features/admin/view_services.feature

```diff
@@ -0,0 +1,26 @@
+@view-a-service
+Feature: View a service
+
+Scenario Outline: Correct user can access the service page
+  Given I am logged in as the production <role> user
+  # TODO GET ME HERE FROM THE LOGIN
+  And I am on the /admin/services/1123456789012346 page
```

Comment:
How do we know this exists? Would be nice to not rely on pre-existing data, because in theory the data could change at any point.  The tests should set up everything they need to start with - this is why we made the old tests - which relied on pre-existing data - "legacy".